myblog/blog/views.py

```diff
@@ -57,9 +59,23 @@ class PostViewSet(ModelViewSet):
     
     authentication_classes = [BasicAuthentication]
     permission_classes = [IsAuthenticated]
-        
+         
     def perform_create(self, serializer):
         serializer.save(author = self.request.user)
+        
+    def get_queryset(self):
+        return BlogPost.objects.filter(author = self.request.user).order_by('title').all()
```

Comment:
I think that queryset for other my-tags should be different because you need the tags where the user was tagged, not where the user is the author.

remember that you can return different queryset depending on the action using `self.action` attribute.